locales/en-US/translation.yml

```diff
@@ -229,67 +229,88 @@ search:
     results: '{{value}} results'
 
 admin:
+  top:
+    title: Wiki Admin
+    description: This page can only be accessed by the Wiki administrators.
+    appoint_admin: You can appoint a user as a Wiki administrator using the Make Admin button in Manage User page.
```

Comment:
Suggested change: You can appoint any user as a Wiki administrator using the Make Admin button in the Manage User page.